frontend/src/vendor/Segment.ts

```diff
@@ -58,8 +58,8 @@ export const initializeSegment = (): void => {
   // for methods in Analytics.js so that you never have to wait
   // for it to load to actually record data. The `method` is
   // stored as the first argument, so we can replay the data.
-  analytics.factory = function(method: any) {
-    return function(...args: any[]) {
+  analytics.factory = function (method: any) {
+    return function (...args: any[]) {
```

Comment:
Nit: can we also revert these changes (since this is a vendored source file)?

Reply:
Not sure what linter decided those were necessary, but something did :) Will remove them

Reply:
Done